rkt/pubkey/pubkey.go

```diff
@@ -234,22 +236,32 @@ func downloadKey(u *url.URL, skipTLSCheck bool) (*os.File, error) {
 	return retTf, nil
 }
 
-func getClient(skipTLSCheck bool) *http.Client {
-	if !skipTLSCheck {
-		return http.DefaultClient
+func newClient(skipTLSCheck bool) *http.Client {
+	dialer := &net.Dialer{
+		Timeout:   30 * time.Second,
+		KeepAlive: 30 * time.Second,
+	} // values taken from stdlib v1.5.3
+
+	tr := &http.Transport{
```

Comment:
That is is a fair argument, rkt being daemon-less though this does not pose a big currently. Although, with mutable CRI daemon-based operations on the horizon (although still not being high traffic), this could be a valid point. I'll refactor this to reuse the transport. Thanks!